librz/core/cmd_descs/cmd_descs.h

```diff
@@ -185,18 +185,29 @@ RZ_IPI RzCmdStatus rz_il_vm_step_with_events_handler(RzCore *core, int argc, con
 RZ_IPI RzCmdStatus rz_il_vm_step_until_addr_handler(RzCore *core, int argc, const char **argv);
 RZ_IPI RzCmdStatus rz_il_vm_status_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode);
 RZ_IPI RzCmdStatus rz_analysis_graph_dataref_handler(RzCore *core, int argc, const char **argv);
+RZ_IPI char **rz_analysis_graph_format_choices(RzCore *core);
```

Comment:
@ret2libc seems generator produces duplicate items in case of `choices_cb`

Reply:
If it does not produce an error right now, I will create an issue and address it in a few days.

Reply:
OK: https://github.com/rizinorg/rizin/issues/2946